packages/at_client/lib/src/at_collection/impl/at_collection_query_operations_impl.dart

```diff
@@ -82,13 +84,16 @@ class AtCollectionQueryOperationsImpl extends AtCollectionQueryOperations {
 
     try {
       AtValue atValue = await getAtClient().get(atKey);
+      if (atValue.value == null) {
+        throw AtKeyNotFoundException('Key has expired');
+      }
```

Comment:
have to actually throw an exception here, since getModel return value is not nullable